lib/core/src/lib/datatable/components/location-cell/location-cell.component.ts

```diff
@@ -18,19 +18,24 @@
 import { ChangeDetectionStrategy, Component, Input, OnInit, Optional, ViewEncapsulation } from '@angular/core';
 import { DataTableCellComponent } from '../datatable-cell/datatable-cell.component';
 import { DataTableService } from '../../services/datatable.service';
+import { AsyncPipe } from '@angular/common';
+import { RouterModule } from '@angular/router';
+import { PathInfo } from '@alfresco/js-api';
 
 @Component({
+    standalone: true,
+    imports: [AsyncPipe, RouterModule],
     selector: 'adf-location-cell',
     changeDetection: ChangeDetectionStrategy.OnPush,
     template: `
         <ng-container>
-            <a href="" [title]="tooltip" [routerLink]="link">
+            <a [title]="tooltip" [routerLink]="link">
                 {{ value$ | async }}
             </a>
         </ng-container>
     `,
     encapsulation: ViewEncapsulation.None,
-    host: { class: 'adf-location-cell adf-datatable-content-cell' }
```

Comment:
don't we need `adf-location-cell` anymore?

Reply:
i've just noticed that we use it inside datatable.component.scss so I will revert it, thx